src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="card card--width" data-qa="card">
+
+        <div class="card__img-container">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__img"
+          >
+        </div>
+
+        <div class="card__content">
+          <div class="card__description">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__product-code">Product code: 195434</div>
```

Comment:
You should use semantic tags! `<p>` fits here. Fix everywhere!
<img width="378" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/ca735e44-5ddf-472d-98fb-60c9e6bf421a">